applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -116,32 +116,32 @@ The test command is not supported.
 Power saving #XSLEEP
 ====================
 
-The ``#XSLEEP`` command makes the nRF9160 development kit enter idle or sleep mode.
+The ``#XSLEEP`` command makes the nRF9160 enter idle or sleep mode.
```

Comment:
```suggestion
The ``#XSLEEP`` command makes the nRF9160 development kit (DK) enter idle or sleep mode.
```

Reply:
Would saying that the command makes the modem sleep be correct since we are talking of the Serial LTE modem application?

Reply:
Not really, modem power on/off is controlled by the app core (AT commands).
What we are talking is nRF9160 SiP sleeps and wake up as a whole.
https://infocenter.nordicsemi.com/topic/ps_nrf9160/chapters/pmu/doc/operationmodes/frontpage.html?cp=2_0_0_4_0

Reply:
```suggestion
The ``#XSLEEP`` command makes the nRF9160 System in Package (SiP) enter idle or sleep mode.
```
And elsewhere where I added DK, use the full "nRF9160 SiP" instead.